src/core/grid_based_algorithms/lbgpu.cuh

```diff
@@ -0,0 +1,25 @@
+#ifndef LBGPU_CUH
```

Comment:
Why do you need a header for this? Just declare it in the cuda file, it's not used anywhere else.

Reply:
it's used in electrokinetics_cuda.cu

Reply:
Ah ok I did not see that.